verification/curator-service/ui/src/components/Users.tsx

```diff
@@ -39,7 +40,9 @@ export default class Users extends React.Component<{}, UsersState> {
         return (
             <div>
                 {this.state.users.map(user =>
-                    <div key={user.googleID}>{user.name}</div>)}
+                    <div key={user.googleID}>
+                        {user.name}: {user.roles?.map(role => `${role}, `)}
```

Comment:
Use join(', ') instead to avoid the last comma?

Reply:
Ah yes of course, done.